src/doctopus/db/schema.clj

```diff
@@ -87,8 +89,20 @@
 (defn- create-fts-document-index
   [db-name]
   (let [update-sql "UPDATE documents SET search_vector = to_tsvector('english', name || ' ' || body)"
-        idx-sql "CREATE INDEX fts_idx ON documents USING GIN(search_vector)"]
-    (log/info "Creating FTS Index for the documents table")
+        idx-sql (->> ["DO $$"
```

Comment:
Ohhh, that is very pleasant! Hadn't met that one before. Might just put that on my to-implement list.